src/commands/init/index.ts

```diff
@@ -79,4 +81,48 @@ export default class Init extends Command {
 
     ux.action.stop();
   }
+
+  private async getStartAppMessage(): Promise<string> {
```

Comment:
You don't need to return `Promise` and use `await` in the code above. Just return regular string ;)

Reply:
changes applied